openfisca_france_local/departements/eure_et_loir/repas_foyer.py

```diff
@@ -0,0 +1,92 @@
+        conditions_aides = condition_apa * condition_aide_menagere_caisse_retraite * condition_aides_actp * condition_aides_mtp
+
+        return condition_residence * condition_age * condition_nationalite * condition_gir * condition_ressources * conditions_aides
+
+
+class eure_et_loir_eligibilite_repas_foyer_personne_handicape(Variable):
+    value_type = bool
+    entity = Individu
+    definition_period = MONTH
+    label = "En Eure-et-Loir,éligibilité à l'aide Repas Foyer de l'Eure-et-Loir pour les personnes handicapées"
+    reference = ["Titre 3 Chapitre 1-2 du règlement départementl d'Aide Sociale PA PH de l'Eure-et-Loir",
+                 "https://github.com/openfisca/openfisca-france-local/wiki/files/departements/eure-et-loir/RDAS_valide__decembre_2019.pdf"
+                 ]
+    documentation = """
+                        Les personnes en situation de handicap peuvent bénéficier d’une prise en charge des frais de repas servis par des foyers restaurant créés par les communes, les CCAS ou les CIAS habilités à l’aide sociale. 
+                        La participation financière du bénéficiaire est déterminée en fonction de ses ressources et du prix du repas.
+                        Cette aide n’est pas cumulable avec l’Allocation compensatrice pour tierce personne (ACTP), la Majoration Tierce Personne (MTP versée par la CPAM) et les prestations d’aide-ménagère servies par les caisses de retraite
+                    """
+
+    def formula(individu, period, parameters):
+        taux_incapacite = individu('taux_incapacite', period)
+        restriction_substantielle_durable = individu('aah_restriction_substantielle_durable_acces_emploi', period)
+        age = individu('age', period)
+        ressortissant_eee = individu('ressortissant_eee', period)
+
+        repas_foyer_parameters = parameters(
+            period).departements.eure_et_loir.repas_foyer.repas_foyer
+
+        individual_resource_names = {
+            'aah',
+            'salaire_imposable',
+            'retraite_imposable',
+            'pensions_invalidite',
+            'revenus_stage_formation_pro'
+        }
+        ressources_famille = {'aspa'}
+        ressources_annuelles = {'retraite_complementaire_artisan_commercant',
+                                'retraite_complementaire_profession_liberale'
+                                }
+        individu_resources_month = sum(
+            sum([individu(resource, period.last_month) for resource in individual_resource_names]),
+            sum([individu.famille(resource, period.last_month) for resource in ressources_famille]))
+        individu_resources = sum(individu_resources_month, sum(
+            [individu(resource, period, options=[DIVIDE]) for resource in ressources_annuelles]))
+
+        condition_residence = individu.menage('eure_et_loir_eligibilite_residence', period)
+        condition_age = age <= repas_foyer_parameters.age_minimal_personne_handicap
```

Comment:
Passer d'inférieur ou égal à strictement inférieur ? 
Le RDAS indique que les conditions d'attributions sont celles de l'article 94 du même document (page 51 du pdf) et il est dit "personnes de moins de 60 ans".
```suggestion
        condition_age = age < repas_foyer_parameters.age_minimal_personne_handicap
```